strapi/strapi
# fix testMatch /ignore

### What does it do?

removes the / in front of testMatch and ignore

### Why is it needed?

Since else I windows users can't run the tests since the / means it will be in root and not current folder

### How to test it?

run tests normally to see I did not break that
and run tests on windows and see that it now finds all the unit tests/ frontend tests
